_data/components/otableData/01tableColumn.yml

```diff
@@ -113,4 +113,10 @@ attributes: [{
   default: "yes",
   required: "",
   description: "Indicates whether or not to resizable the column"
+},{
+  name: "groupable",
+  type: "no | yes | true | false",
+  default: "yes",
+  required: "",
+  description: "Indicates whether or not to groupable the column"
```

Comment:
"groupable" is an adjective.
"Indicates the column is groupable" or "Indicates whether or not the column is groupable".